nfpm.yaml

```diff
@@ -0,0 +1,72 @@
+---
+
+# Name. (required)
+name: backupctl
+
+# Architecture. (required)
+arch: noarch
+
+# Platform.
+# Defaults to `linux`.
+platform: linux
+
+# Version. (required)
+# This will expand any env var you set in the field, eg version: v${SEMVER}
+version: v2.0.4
```

Comment:
Is there any better what than hardcoding the version in the nfpm config? I can tell from personal experience that something like updating this version number gets forgotten a lot. In fact, this version number doesn't match then one in `backupctl/version.py`

Is the version available in some env var? E.g. before running nfpm, you could e.g. run `export BACKUPCTL_VERSION=$(python -c 'import backupctl.version; print(backupctl.version.__version__)')` and pass `-e BACKUPCTL_VERSION` to the docker nfpm run, and put `version: v${BACKUPCTL_VERSION}` here. (UNTESTED)

Reply:
Ive set the version of the package once in the action.yml, i dont like it to be the source of the version, i might need to move it elsewhere, if you have an idea where, im open for ideas.

Reply:
I added a version file with just the version in it and removed the "env" keyword.
Seems more clear to me.